app/workers/digest_email_generation_worker.rb

```diff
@@ -0,0 +1,39 @@
+class DigestEmailGenerationWorker
+  include Sidekiq::Worker
+
+  def perform(subscriber_id:, digest_run_id:)
```

Comment:
This should be taken a DigestRunSubscriber, do they exist yet?

Reply:
No, not yet. I'll do that in the next PR when we connect this up to the initiator thing